### PR TITLE
Add new service Logs Router to IBM Terraform Provider

### PR DESCRIPTION
**Pull request by ianre**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccIbmLogsRouter'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test $(go list ./... |grep -v 'vendor') -v -run=TestAccIbmLogsRouter -timeout 700m
?   	github.com/IBM-Cloud/terraform-provider-ibm	[no test files]
?   	github.com/IBM-Cloud/terraform-provider-ibm/ibm/acctest	[no test files]
?   	github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex	[no test files]
?   	github.com/IBM-Cloud/terraform-provider-ibm/ibm/provider	[no test files]
testing: warning: no tests to run
PASS
ok  	github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns	(cached) [no tests to run]
[WARN] Set the environment variable IBM_PROJECTS_CONFIG_APIKEY for testing IBM Projects Config resources, the tests will fail if this is not set
... [Many warnings for unset variables for other services' acceptance tests]
[WARN] Set the environment variable IBM_SATELLITE_SSH_PUB_KEY with a ssh public key or ibm_satellite_* tests may fail
=== RUN   TestAccIbmLogsRouterTenantDataSourceBasic
--- PASS: TestAccIbmLogsRouterTenantDataSourceBasic (36.69s)
=== RUN   TestAccIbmLogsRouterTenantBasic
--- PASS: TestAccIbmLogsRouterTenantBasic (31.64s)
PASS
ok  	github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/logsrouter	(cached)
[WARN] Set the environment variable IBM_PROJECTS_CONFIG_APIKEY for testing IBM Projects Config resources, the tests will fail if this is not set
... [Many warnings for unset variables for other services' acceptance tests]
[WARN] Set the environment variable IBM_SATELLITE_SSH_PUB_KEY with a ssh public key or ibm_satellite_* tests may fail
testing: warning: no tests to run
PASS
ok  	github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc	(cached) [no tests to run]

```

### CODE REVIEW

File: ibm/conns/config.go [hunk 1]
```diff
@@ -1169,6 +1175,11 @@ func (session clientSession) MetricsRouterV3() (*metricsrouterv3.MetricsRouterV3
 	return session.metricsRouterClient, session.metricsRouterClientErr
 }
 
+// Logs Router API
+func (session clientSession) IbmLogsRouterOpenApi30V0() (*ibmlogsrouteropenapi30v0.IbmLogsRouterOpenApi30V0, error) {
+	return session.ibmLogsRouterOpenApi30Client, session.ibmLogsRouterOpenApi30ClientErr
+}
+
```
Comment (kavya498) [inline ibm/conns/config.go:1182]: Add emptycredentialsError check for logs router here - https://github.com/IBM-Cloud/terraform-provider-ibm/blob/a81d5798b7a5a27b50d740633da3ec1b9851ed93/ibm/conns/config.go#L1310

File: website/docs/d/logs_router_tenant.html.markdown [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_logs_router_tenant"
+description: |-
+  Get information about logs_router_tenant
+subcategory: "IBM Logs Router"
```
Comment (kavya498) [inline website/docs/d/logs_router_tenant.html.markdown:6]: subcategory should be `Logs Router`. Kindly update in resource docs as well.

File: website/docs/r/logs_router_tenant.html.markdown [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+* `id` - The unique identifier of the logs_router_tenant.
+* `account_id` - (String) The account ID the tenant belongs to.
+  * Constraints: The maximum length is `32` characters. The minimum length is `32` characters. The value must match regular expression `/[a-z,A-Z,0-9,-]/`.
+* `created_at` - (String) Time stamp the tenant was originally created.
+  * Constraints: The maximum length is `36` characters. The minimum length is `1` character. The value must match regular expression `/[0-9,:,.,-,T,Z]/`.
+* `updated_at` - (String) time stamp the tenant was last updated.
+  * Constraints: The maximum length is `36` characters. The minimum length is `1` character. The value must match regular expression `/[0-9,:,.,-,T,Z]/`.
+
+## Provider Configuration
```
Comment (kavya498) [inline website/docs/r/logs_router_tenant.html.markdown:48]: Kindly remove `Provider Configuration`, `Static credentials` and `Environment variables` sections as they are redundant here.
This is generic info and is available in provider docs.

**Review by kavya498 — CHANGES_REQUESTED** [review]

File: ibm/conns/config.go (hunk 1 above)
Comment (ianre) [reply ibm/conns/config.go:1182]: Added emptycredentialsError check for a Logs Router Client below: https://github.com/IBM-Cloud/terraform-provider-ibm/pull/4945/files#diff-971fca808334dec2e3c5f2462c1d78588f11e0feb1e4fc418dc652fda857511fR1310

File: website/docs/d/logs_router_tenant.html.markdown (hunk 2 above)
Comment (ianre) [reply website/docs/d/logs_router_tenant.html.markdown]: Changed Subcategory to "Logs Router"

File: website/docs/r/logs_router_tenant.html.markdown (hunk 3 above)
Comment (ianre) [reply website/docs/r/logs_router_tenant.html.markdown]: Changed Subcategory to "Logs Router", and removed the redundant general provider docs

**Review by kavya498 — APPROVED** [review]

**Pull request merged**